### PR TITLE
working group: add vectorized expression working group

### PR DESCRIPTION
**Pull request by zz-jason**
as the title says

### CODE REVIEW

**Comment by zz-jason** [comment]
@gingerkidney @qw4990 PTAL

File: working-groups/wg-vec-expr.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Vectorizing all the builtin functions. You can find the function list in [tidb
+issue/12058](https://github.com/pingcap/tidb/issues/12058).
+
+## Meetings
+
+* Regular Working Group Meeting: Wednesdays at 19:30 PT (Pacific Time)
+  (biweekly).  [Convert to your
+  timezone](http://www.thetimezoneconverter.com/?t=19:30&tz=PT%20%28Pacific%20Time%29).
+* Meeting Zoom: (TODO: add a zoom link)
```
Comment (qw4990) [inline working-groups/wg-vec-expr.md]:
```suggestion
* Meeting Zoom: https://pingcap.zoom.us/j/3190482744
```

File: working-groups/wg-vec-expr.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* Meeting Notes: (TODO: add a google doc link)
```
Comment (qw4990) [inline working-groups/wg-vec-expr.md]:
```suggestion
* [Meeting Notes and Agenda](https://docs.google.com/document/d/1bHZs-qj91jH0crUFgROQ0P076rosDQNm2CsUTvDr0zo/edit?usp=sharing)
```

File: working-groups/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Working Groups
+
+Working Groups provide a formal avenue for disparate groups to collaborate
+around a common problem, craft a balanced position, and disband. Because they
```
Comment (dcalvin) [inline working-groups/README.md]: "disband" should not be a goal

Comment (dcalvin) [inline working-groups/README.md]:
```suggestion
around a common problem, craft a balanced position, and disband. A working group
```

File: working-groups/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+represent the interests of multiple groups, they are a vehicle for consensus
```
Comment (dcalvin) [inline working-groups/README.md]:
```suggestion
represents the interest of the group and serves as a vehicle for a consensus
```

File: working-groups/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+building. If code is developed as part of collaboration within the Working
```
Comment (dcalvin) [inline working-groups/README.md]:
```suggestion
building. If the code is developed as part of the collaboration within the Working
```

File: working-groups/README.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+Group, that code will be housed in an appropriate repository as described in
+the repositories document. The merging of this code into the repository will be
```
Comment (dcalvin) [inline working-groups/README.md]:
```suggestion
the repository document. Merging of this code into the repository will be
```

File: working-groups/README.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+governed by the standard policies regarding submitting code to that repository
```
Comment (dcalvin) [inline working-groups/README.md]:
```suggestion
governed by the related standard policies.
```

File: working-groups/wg-vec-expr.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+# Vectorized Expression Working Group
+
+## Goals
+
+Vectorizing all the builtin functions. You can find the function list in [tidb
+issue/12058](https://github.com/pingcap/tidb/issues/12058).
+
+## Meetings
+
+* Regular Working Group Meeting: Wednesdays at 19:30 PT (Pacific Time) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=19:30&tz=PT%20%28Pacific%20Time%29).
```
Comment (dcalvin) [inline working-groups/wg-vec-expr.md]:
```suggestion
* Regular Working Group Meeting: 19:30, Wednesdays at PT (Pacific Time) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=19:30&tz=PT%20%28Pacific%20Time%29).
```

File: working-groups/wg-vec-expr.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+* Meeting Zoom: [https://pingcap.zoom.us/j/3190482744](https://pingcap.zoom.us/j/3190482744)
+* Meeting Notes: [Meeting Notes and Agenda](https://docs.google.com/document/d/1bHZs-qj91jH0crUFgROQ0P076rosDQNm2CsUTvDr0zo/edit)
+
+## Organizers
```
Comment (dcalvin) [inline working-groups/wg-vec-expr.md]:
```suggestion
## Organizer
```

File: working-groups/README.md (hunk 3 above)
Comment (zz-jason) [reply working-groups/README.md]: refactored, PTAL

**Comment by gingerkidney** [comment]
LGTM

**Pull request merged**